pipelines/controller/config/application.yaml

```diff
@@ -97,12 +97,11 @@ fhirdata:
   # https://github.com/google/fhir-data-pipes/issues/785.
   resourceList: "Patient,Encounter,Observation,Questionnaire,Condition,Practitioner,Location,Organization"
 
-  # The maximum number of Flink workers to use. Each worker may have 
-  # `numThreads` parallel threads.
-  maxWorkers: 1
-
-  # If a positive number, the number of threads to use per worker.
-  # Otherwise, the number of threads is equal to the number of cores.
+  # The parallelism to be used for a pipeline job. In case of FlinkRunner, if the value is set to
+  # -1, then in the local execution mode the number of threads the job uses will be equal to the
+  # number of cores in the machine, whereas in the remote mode (cluster) only 1 thread is used.
```

Comment:
Question: In the cluster mode, when you say "only 1 thread is used" you mean one thread per worker? Or is it one thread and one worker? And how can we set the number of workers?

Reply:
If a Flink job is submitted to the cluster with parallelism of 1 (or default parallelism), then it is executed with one of the Taskmanager slot that is available among all the slots across the cluster, more details [here](https://nightlies.apache.org/flink/flink-docs-release-1.2/setup/config.html#configuring-taskmanager-processing-slots). I am not sure on how we can set the number of workers, I am assuming in earlier versions of Flink it is a static config and in the latest version it is determined based on the Parallelism needed and the slots available for TaskManager i.e `#Workers = Parallelism / Slots per TaskManager`

Reply:
Thanks @chandrashekar-s these information details are very helpful. Please consider adding a summary in issue #893 such that we later address it when it is needed.